leetcode/numbers_1.rs

```diff
@@ -0,0 +1,37 @@
+//Minimum Sum of Four Digit Number
+pub fn minimum_sum(mut num: i32) -> i32 {
+    let mut fullvec = Vec::with_capacity(4);
+
+    (0..4).for_each(|_| {
+        fullvec.push(num % 10);
+        num /= 10;
+    });
+
+    fullvec.sort();
+
+    if let &[a1, a2, a3, a4] = fullvec.as_slice() {
+        return 10 * a1 + 10 * a2 + a3 + a4;
+    
+    unreachable!()
+    }
+}
+
+// Number of Good Pairs
+pub fn num_identical_pairs(nums: Vec<i32>) -> i32 {
+    let mut pairs= 0;
+    
+    for i in 0..nums.len() {
+        for j in i+1..nums.len() {
+            if nums[i]==nums[j] {
+            pairs+=1;
```

Comment:
Run rustfmt before you commit. You may want to turn on "Format on Save" in your VS Code settings so that you don't have to worry about formatting any more.